js/fileexplorer/shared/FileTree.svelte

```diff
@@ -108,11 +127,27 @@
 						<img src={name === "." ? FolderIcon : FileIcon} alt="file icon" />
 					</span>
 				{/if}
-				{name}
+				<span
+					class="item-name"
+					class:selectable
+					role={selectable ? "button" : undefined}
+					tabindex={selectable ? 0 : undefined}
+					on:click|stopPropagation={() => {
+						if (selectable) {
+							handle_select([...index_path, i], [...path, name], type);
```

Comment:
If the file explorer is interactive, clicking on the textbox does not trigger the select event. Only clicking on the name triggers the select event. which is not intuitive